sky/provision/vast/instance.py

```diff
@@ -89,6 +89,15 @@ def run_instances(region: str, cluster_name: str, cluster_name_on_cloud: str,
                                           resumed_instance_ids=[],
                                           created_instance_ids=[])
 
+        docker_login_params: Optional[str] = None
+        docker_login_config = config.node_config.get('DockerLogin')
```

Comment:
wait why not use `docker_login_config`?

Reply:
also I think for other clouds, we have some custom logic (like adding `<server>/...` for private registries) to deal with this docker config. Can you take a look at those?